Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity.yml

```diff
@@ -762,17 +774,857 @@ script:
+      isArray: false
+      name: case_id
+      required: true
+    - description: A comma seperated list of custodians ids to apply a hold to.
+      isArray: true
+      name: custodian_id
+      required: true
+    description: |-
+      Start the process of applying hold on eDiscovery custodians.
+      Available return statuses: 
+      notApplied - The custodian is not on Hold (all sources in it are not on hold).
+      applied - The custodian is on Hold (all sources are on hold).
+      applying - The custodian is in applying hold state (applyHold operation triggered).
+      removing - The custodian is in removing the hold state(removeHold operation triggered).
+      partial - The custodian is in mixed state where some sources are on hold and some not on hold or error state.
+    execution: false
+    name: msg-apply-hold-ediscovery-custodian
+  - arguments:
+    - description: The id of the eDiscovery case.
+      isArray: false
+      name: case_id
+      required: true
+    - description: A comma seperated list of custodians ids to remove a hold from.
+      isArray: true
+      name: custodian_id
+      required: true
+    description: Start the process of removing hold from eDiscovery custodians.
+    execution: false
+    name: msg-remove-hold-ediscovery-custodian
+  - arguments:
+    - description: The id of the eDiscovery case.
+      isArray: false
+      name: case_id
+      required: true
+    - description: The display name of the search.
+      isArray: false
+      name: display_name
+      required: true
+    - description: The description of the search.
+      isArray: false
+      name: description
+      required: false
+    - description: The query string used for the search. The query string in KQL (Keyword Query Language) format. For details, see Keyword queries and search conditions for Content Search and eDiscovery. You can refine searches by using fields paired with values; for example, subject:"Quarterly Financials" AND Date>=06/01/2016 AND Date<=07/01/2016.
+      isArray: false
+      name: content_query
+      required: false
+    - description: When specified, the collection will span across a service for an entire workload.
+      isArray: false
+      name: data_source_scopes
+      required: false
+      auto: PREDEFINED
+      predefined:
+      - none
+      - allTenantMailboxes
+      - allTenantSites
+      - allCaseCustodians
+      - allCaseNoncustodialDataSources
+    description: Create a new eDiscoverySearch object.
+    execution: false
+    name: msg-create-ediscovery-search
+    outputs:
+    - contextPath: MsGraph.eDiscoverySearch.ContentQuery
+      description: 'The query string in KQL (Keyword Query Language) query. For details, see Keyword queries and search conditions for Content Search and eDiscovery. You can refine searches by using fields paired with values; for example, subject:"Quarterly Financials" AND Date>=06/01/2016 AND Date<=07/01/2016.'
```

Comment:
Where is this:  Keyword queries and search conditions for Content Search and eDiscovery?  Need a link or remove.

Reply:
added the link